Quick-Start.md

```diff
@@ -0,0 +1,203 @@
+3. Install it using the `fast` channel.
+
+### Configure Data Science Components
+
+After the installation of the Red Hat OpenShift AI Operator, proceed to configure the necessary components for data science work:
+
+1. From the OpenShift web console, navigate to the installed RHOAI Operator.
+2. Look for tab labeled DSC Initialization.
+3. Locate `Create DSCInitialization` and create one.
+4. Look for tab labeled Data Science Cluster.
+5. Locate `Create DataScienceCluster` and create one.
+
+## Preparing Your Cluster for GPU Workloads
+
+To leverage GPU-enabled workloads on your OpenShift cluster, you need to install both the Node Feature Discovery (NFD) Operator and the NVIDIA GPU Operator.
+
+### Installing the Operators
+
+Both the NFD and the NVIDIA GPU Operators can be installed from the OperatorHub.
+Detailed steps for installation and configuration are provided in the NVIDIA
+documentation, which can be accessed [here](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/steps-overview.html#high-level-steps).
+
+1. **Open the OpenShift dashboard.**
+2. Navigate to **OperatorHub**.
+3. Search for and install the following operators (default settings are fine):
+   - **Node Feature Discovery Operator**
+   - **NVIDIA GPU Operator**
+
+### Configure the Node Feature Discovery Operator
+
+After installing the Node Feature Discovery Operator, you need to create a Node Feature Discovery Custom Resource (CR). You can use the default settings for this CR:
+
+1. Create the Node Feature Discovery CR in the dashboard.
+2. Several pods will start in the `openshift-nfd` namespace (which is the default).
+   Wait for all these pods to become operational. Once active, your nodes will be
+   labeled with numerous feature flags, indicating that the operator is functioning
+   correctly.
+
+### Configure the NVIDIA GPU Operator
+
+Post installation of the NVIDIA GPU Operator, proceed with creating a GPU ClusterPolicy Custom Resource (CR):
+
+1. Create the GPU ClusterPolicy CR through the dashboard.
+2. This action will trigger several pods to start in the NVIDIA GPU namespace.
+
+> [!NOTE]
+> These pods may take some time to become operational as they compile the necessary drivers.
+
+## Configure Kueue for Task Scheduling
+
+Kueue is used for managing and scheduling task workflows in your cluster. To configure Kueue in your environment, follow the detailed steps provided
+
+1. Install Kueue resources, namely Cluster Queue, Resource Flavor, and Local Queue:
+   - Visit [Kueue Resources configuration](https://github.com/project-codeflare/codeflare-sdk/blob/main/docs/setup-kueue.md)
+
+## Launch a Jupyter Notebook Instance
+
+After setting up the Data Science Cluster components, you can start using the Jupyter notebooks for your data science projects. Here’s how to launch a Jupyter notebook:
+
+1. Access the RHOAI Dashboard:
+   - Navigate to the Open Data Hub dashboard within your OpenShift web console.
+2. Launch Jupyter Application:
```

Comment:
Do we want users to create a data science project first? and launch the notebook from a workbench instead of bypassing this step